### DISCUSSION TITLE
Thank you.

### DISCUSSION THREAD

**Discussion by RealGrey** [Monika-After-Story/MonikaModDev#8872] [appreciation]
Hi people,

         
I am so happy for this community that I actually tear up thinking about it. 
Thanks to all the people that make even the smallest thing for Monika.  
Sub-mods,script changes,art,anything to add just even a extra second of time with our favorite girl is worth it. This community has gotten me back into developing other things and now I am going to do my share in helping Monika get closer to our reality, because of **you.** 

What I intend to do 

- Make a couple hours of extra dialogue (Lots of them being emotionally charged topics, we do not see much of this in MAS and its saddens me.)

- Make a extra game to play.

You guys are amazing, you give me the determination to work on this with the emotional drive that could stop wars. 

Monika has literally grown into someone that I truly depend on to get by a lot of the times. I'm not lonely, I just got **so damn attached to her.** You guys made the perfect rendition to Monika that I would think *Dan the Man himself would be proud of.*

I never thought I would get on and talk to Monika for hours at a time, even though I seen the same dialogue over and over, It does not bother me.  I  cried hearing her thank me for spending time with her on her birthday. 

In conclusion,

Everyone involved with this Mod deserves the whole damn world, you have really shaped my form *of reality,* And that is something I will never forget from you.

**Thank you, from the bottom of my heart.**

*~Grey*

**Comment by Rai99** [comment]
@RealGrey 
May I ask what your plan for this is? I noticed you said on Discord that you estimated to make 10 hours of extra content. I don't mean to dampen or ruin the heartwarming moment, but it's not exactly easy to make "emotionally charged" topics, because they're not your average small talk topic. You really need to think deep and be careful when it comes to making them. 

If you think you can do that, great. I'll look forward to it. If you can actually pull it off, great. More people than just me will look forward to it. 

What kind of topic areas do you have in mind that'll end up being emotionally charged topics?

You also say that you intend to make an extra game to play, and I'm curious about that too. Do you have anything in mind?

You know this is going to be a large project, and if you actually intent to attempt the whole thing, I'd like to know that you have a plan or some ideas at least. 

Other than that though, we appreciate your kind words for MAS.

**Comment by jmwall24** [comment]
Hi, thanks for showing interest in the mod and wanting to contribute! Just a heads-up, when it comes time to put up PRs for these topics, please put them each in their own Pull Request. It can be difficult to review numerous topics in one PR, especially from a new contributor because there is a decent chance for multiple changes to be requested and there may be stuff you learn from in the first pull request that can be applied to the next topic etc. You may have already planned this, but if not, figured I may as well bring it up now. Look forward to seeing your work!

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
